framework_tvb/tvb/interfaces/web/entities/context_selected_adapter.py

```diff
@@ -44,14 +44,25 @@ class SelectedAdapterContext(object):
     """
     
     KEY_CURRENT_ADAPTER_INFO = "currentAdapterInfo"
-    _KEY_INPUT_TREE = "inputList"
     _KEY_CURRENT_STEP = "currentStepCategoryId"
     _KEY_CURRENT_SUBSTEP = "currentAlgoGroupId"
     _KEY_SELECTED_DATA = 'defaultData'
     KEY_PORTLET_CONFIGURATION = 'portletConfig'
-    KEY_TREE_DEFAULT = "defaultTree"
+    KEY_VIEW_MODEL = "viewModel"
```

Comment:
code related to portletConfig can also be cleaned

Reply:
This context has become now very simple (only one key) but I still think it is better to keep it as a "context", and we should reuse the concept for grouping simulation keys also.